FHEM/00_SIGNALduino.pm

```diff
@@ -5494,6 +5500,12 @@ USB-connected devices (SIGNALduino):<br>
       <li>SlowRF<br>
         Modulation ASK/OOK, <b>l&auml;d die Standard Einstellung vom uC</b>
       </li>
+      <li>WMBus_S<br>
+        Modulation FSK, Datenrate=32.7301 kbps, Sync Word=7696, Frequenz 868.300 MHz (Verarbeitung muss noch in Firmware integriert werden)
```

Comment:
Die Datenrate ist hier eigentlich genauer: 32.768 kbaud
Das sich das beim CC110x evtl. nicht so genau einstellen lässt, ist eine andere Sache.

Reply:
Naja, mag ja sein - ich würde aber lieber die Werte angeben, die in der Norm stehen.